.github/workflows/test.yml

```diff
@@ -91,19 +91,11 @@ jobs:
           VERCEL_TEST_TOKEN: ${{ secrets.VERCEL_TEST_TOKEN }}
           VERCEL_TEST_REGISTRATION_URL: ${{ secrets.VERCEL_TEST_REGISTRATION_URL }}
           FORCE_COLOR: '1'
-      - name: 'Determing Turbo HIT or MISS'
-        id: turbo-summary
-        shell: bash
-        run: |
-          TURBO_MISS_COUNT=`node utils/determine-turbo-hit-or-miss.js`
```

Comment:
@trek Lets instead keep the script but remove process.exit()

Reply:
I'm not in a good place to switch contexts. I can do a full revert, or revert of the workflow file.